:indexed transcoder hint support

hanshuebner opened the pull request:
This change adds a new `:indexed` transcoder hint option for PNG files which can be used to generate non-truecolor images.  To support this option, I had to add support for non-float transcoder hints by attaching a coercing function to each defined hint.

Also, a .gitignore file has been added.

pallix commented:
Thanks Hans :-).

There are some tests for the transcoder, are they still passing? I don't have much time to check.

hanshuebner commented:
Hi Pierre,

the transcoder tests, but there was a deliberately failing test in core.clj
which I have now removed.  I've also added a primitive test to verify the
operation of the :indexed option.

Thanks,
Hans

2015-10-13 11:00 GMT+02:00 Pierre Allix <notifications@github.com>:

> Thanks Hans :-).
>
> There are some tests for the transcoder, are they still passing? I don't
> have much time to check.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pallix/tikkba/pull/8#issuecomment-147653538>.
>

pallix commented:
Thank you!

pallix commented:
I bumped the version on Clojars.
